miden/tests/integration/stdlib/crypto/rpo.rs

```diff
@@ -0,0 +1,148 @@
+use super::build_test;
+use crate::helpers::build_expected_hash;
+use vm_core::StarkField;
+
+#[test]
+fn test_hash_empty() {
+    // when the address range contains zero elements, the result is all zeros
+    let empty_range = "
+    use.std::crypto::hashes::rpo
+
+    begin
+        push.1000 # end address
+        push.1000 # start address
```

Comment:
Should this fail?

Reply:
I think this is fine. I think I saw a comment somewhere that said that start address must be smaller than end address - if that's the case, we should modify that comment to say if the address are the same this just computes a hash of a single word.

Reply:
done together w/ https://github.com/0xPolygonMiden/miden-vm/pull/805#discussion_r1152648652